src/view/com/util/forms/PostDropdownBtn.tsx

```diff
@@ -210,6 +212,20 @@ let PostDropdownBtn = ({
         web: 'comment-slash',
       },
     },
+    hasSession && {
+      label: _(msg`Mute words & tags`),
+      onPress() {
+        mutedWordsDialogControl.open()
+      },
+      testID: 'postDropdownMuteWordsBtn',
+      icon: {
+        ios: {
+          name: 'speaker.slash',
```

Comment:
Mmm I don't think there's a funnel SFSymbol...Not sure what we'd use for this rather than the speaker slash. There's some options, but they don't like up with what we have for icons in the pack :(